_partials/self-hosted/_aws-sts-config.mdx

```diff
@@ -0,0 +1,96 @@
+---
+partial_category: self-hosted
+partial_name: sts-config
+---
+
+{props.edition} allows tenants to use AWS Secret Token Service (STS) to add AWS cloud accounts to their {props.edition} environment. 
+In order to do this, {props.edition} relies on another AWS cloud account that is configured on the {props.edition} instance 
+to assume a custom role created on the tenant's AWS account. 
+
+This custom role establishes a trust relationship between the tenant's AWS account and the AWS account bound to the {props.edition} instance. 
+It allows the {props.edition}-bound AWS account to receive temporary credentials from the tenant's AWS account to deploy infrastructure in AWS using STS. For more information on how to add an AWS cloud account to a tenant using STS, refer to the  <VersionedLink text="Add an AWS Account to Palette" url="/clusters/public-cloud/aws/add-aws-accounts/" /> guide.
+
+You must configure your {props.edition} instance with an STS AWS account before your tenants can add AWS cloud accounts 
+to deploy clusters in AWS using STS. Without this configuration, the STS option will be greyed out when your tenants try to add an AWS account. 
+
+## Prerequisites
+
+- Access to the {props.edition} system console.
+
+- The role of Root Administrator or Account Administrator.
+
+- An AWS IAM user that is allowed to assume cross-account IAM roles. 
+
+## Enable Adding AWS Accounts Using STS
+
+1. Open a terminal session.
+
+2. Log in to the {props.edition} System API by using the `/v1/auth/syslogin` endpoint. 
+   Use the curl command below and replace the `example.com` placeholder URL with the URL of your {props.edition} instance. 
+   Ensure you replace the credentials below with your system console credentials.
+
+   ```shell
+   TOKEN=$(curl --insecure --location 'https://example.com/v1/auth/syslogin' \
+    --header 'Content-Type: application/json' \
+    --data '{
+    "password": "**********",
+    "username": "**********"
+    }')
+   ```
+
+3. Next, prepare a payload for the AWS account you want to configure. 
+   Use the following JSON payload as a template and replace the `accessKey`, `secretKey`, and `accountId` fields with the AWS access key, secret key, and account ID of your AWS account.
+
+    ```json
+    CONFIG_JSON=$(cat <<EOF
+    {
+    "accessKey": "**********",
+    "secretKey": "**********",
+    "accountId": "123456789"
```

Comment:
Indentation is off

Reply:
I tried to add indentation, but they actually mess up the command so I think I will just leave it this way